app/controllers/application_controller.rb

```diff
@@ -5,6 +5,140 @@ class ApplicationController < Sinatra::Base
   configure do
     set :public_folder, 'public'
     set :views, 'app/views'
+    enable :sessions
+    set :session_secret, "secret"
   end
 
-end
\ No newline at end of file
+  get '/' do
+    @session = session
+    @tweets = Tweet.all
+    erb :index
+  end
+
+  ################### users ###################
```

Comment:
These comments are nice, but...why not just put the actions in a separate controller (i.e. `users_controller` and `tweets_controller`)?